docs/prepare-cdot-clusters.md

```diff
@@ -49,144 +42,100 @@ Choose a role name (e.g. *harvest2-role*). In the *REST API PATH* field, type */
-that we just created. Under *User Login Methods* select *ONTAPI*, and select one of the two authentication methods. Type
-in a password if you chose *Password*. Click on *Save*
+that we just created. Under *User Login Methods* select *ONTAPI*, and one of the two authentication methods.
+Press the `Add` button and select *HTTP* and one of the authentication methods.
+Type in a password if you chose *Password*. Click on *Save*
 
 ![System Manager Settings](assets/prepare-ontap/ontap_user_sm_3.png)
 
 If you chose *Password*, you can add the username and password to the Harvest configuration file and start Harvest. If
 you chose *Certificate* jump to [Using Certificate Authentication](#using-certificate-authentication) to generate
 certificates files.
 
-### System Manager: Classic interface
-
-Open System Manager. Click on the Settings icon in the top-right corner of the window.
-
-![System Manager Classic Settings](assets/prepare-ontap/ontap_user_smc_0.png)
-
-Click on *Roles* in the left menu bar and click *Add*. Choose a role name (e.g. *harvest2-role*).
-
-![System Manager Classic Settings](assets/prepare-ontap/ontap_user_smc_1.png)
-
-If you want to give Harvest read-only access to **all** API objects, then under *Role Attributes* click on *Add*, under
-*Command* type *DEFAULT*, leave *Query* empty, select *readonly* under *Access Level*, click on *OK* and *Add*.
-
-If you want to limit the API objects, then under *Role Attributes*, add each of the following lines as an entry. All of
-those should be entered under the *Command* column, *Query* should be left blank, and *Access Level* should be selected
-*readonly*.
-
-* cluster
-* qos
-* lun
-* snapmirror
-* statistics
-* storage aggregate
-* storage disk
-* storage shelf
-* system node
-* version
-* volume
-
-After you click on *Add*, this is what you should see:
-
-![System Manager Classic Settings](assets/prepare-ontap/ontap_user_smc_2.png)
-
-Now we need to create a user. Click on *Users* in the left menu bar and *Add*. Choose a username and password. Under
-*User Login Methods*, click on *Add*, select *ontapi* as *Application* and select the role that we just created as
-*Role*. Click on *Add* in the pop-up window to save.
-
-![System Manager Classic Settings](assets/prepare-ontap/ontap_user_smc_3.png)
-
-Now add the username and password to `harvest.yml` and start Harvest.
+??? "System Manager Classic interface"
+
+    Open System Manager. Click on the Settings icon in the top-right corner of the window.
+    
+    ![System Manager Classic Settings](assets/prepare-ontap/ontap_user_smc_0.png)
+    
+    Click on *Roles* in the left menu bar and click *Add*. Choose a role name (e.g. *harvest2-role*).
+    
+    ![System Manager Classic Settings](assets/prepare-ontap/ontap_user_smc_1.png)
+    
+    Under *Role Attributes* click on *Add*, under *Command* type *DEFAULT*, leave *Query* empty, select *readonly* under *Access Level*, click on *OK* and *Add*.
+    
```

Comment:
Seems SM images are not populating for me in this section, above section shows images as-is.
<img width="1532" alt="image" src="https://github.com/NetApp/harvest/assets/83282894/9cbcbcb1-dca7-4738-9972-5c91fc2fc23b">